src/components/SecondaryPanes/Scopes.js

```diff
@@ -112,12 +137,27 @@ export default connect(
       selectedFrame && selectedFrame.id
     ) || { scope: null, pending: false };
 
+    const {
+      scope: generatedFrameScopes,
+      pending: generatedPending
+    } = getGeneratedFrameScope(state, selectedFrame && selectedFrame.id) || {
+      scope: null,
+      pending: false
+    };
+
+    const isLoading = generatedPending || pending;
+
     return {
       selectedFrame,
       isPaused: getIsPaused(state),
-      isLoading: pending,
+      isLoading,
       why: getPauseReason(state),
-      frameScopes: frameScopes
+      frameScopes,
+      generatedFrameScopes,
+      hasOriginalScopes:
+        isLoading || !generatedFrameScopes || !frameScopes
+          ? false
+          : generatedFrameScopes.actor !== frameScopes.actor
```

Comment:
hmm, this seems like something that we should have in `reducers/pause` so that the mochitests can also listen for it to